src/server/scripts/Commands/cs_reload.cpp

```diff
@@ -856,7 +856,10 @@ class reload_commandscript : public CommandScript
         LOG_INFO("server.loading", "Re-Loading Page Texts...");
         sObjectMgr->LoadPageTexts();
         handler->SendGlobalGMSysMessage("DB table `page_texts` reloaded.");
-        handler->GetSession()->SendNotification("You need to delete your client cache or change the cache number in config in order for your players see the changes.");
+        if (handler->GetSession())
+        {
+            handler->GetSession()->SendNotification("You need to delete your client cache or change the cache number in config in order for your players see the changes.");
```

Comment:
Isnt there one for PSendNotification or similar instead that can be used?

Reply:
As fas as I know there is no PSendNotification or similar, this function already does the formatting tho.

Reply:
Maybe just use `SendGlobalGMSysMessage`?

Reply:
You can use `PSendSysMessage`, I believe it only sends to players

Reply:
Why send this message to the players? Players should not see this kind of message, this command is for the gm who is fixing something, he can execute it several times

Reply:
Yes, but a player still needs to delete client cache, and I know Barbz wanted it that way. but for me idc, I don't mind either

Reply:
I would make sense to send to all players online at the time, however that would still not fix the problem, players offline? Not paying attention to chat ... some kind of announcement would need to be made over several days to get everyone to delete the cache.
I would keep this for GM only, so as a GM you can debug it and once you done make an announcement for all to do so.
`SendGlobalGMSysMessage` does make sense the notification shows a text on screen while this will show it in gm chat and should work fine. It depends on what is the expectation of this output really if it needs to be a notification or not, I do like GMSysMessage better tbh

Reply:
use it

Reply:
Also on a side note, as the message mentioned you can change the server ClientCacheVersion = 0 to force a cache reset on all clients.